pravega/pravega-operator
# Issue 64: configure bookie metrics

### Change log description
- Enable bookkeeper metrics configuration.
- The user can decide whether to turn on bookkeeper metrics and to choose metrics provider by specifying it in Kubernetes manifest. 

### Purpose of the change
Fix #63 
Fix #116

### How to test the change
A possible way to test: choose a stats provider like Prometheus, deploy Prometheus in k8s and port forward it to local, then use a browser to see if there are any metrics. 
Another way: choose codahale and specify a [csv endpoint](https://github.com/apache/bookkeeper/blob/master/conf/bk_server.conf#L800), use a script to check if there are any files created.
